client/ayon_deadline/plugins/publish/max/submit_max_deadline.py

```diff
@@ -88,57 +47,10 @@ def get_job_info(self):
         )
         job_info.Frames = frames
 
-        job_info.Pool = instance.data.get("primaryPool")
-        job_info.SecondaryPool = instance.data.get("secondaryPool")
-
-        attr_values = self.get_attr_values_from_data(instance.data)
-
-        job_info.ChunkSize = attr_values.get("chunkSize", 1)
-        job_info.Comment = context.data.get("comment")
-        job_info.Priority = attr_values.get("priority", self.priority)
-        job_info.Group = attr_values.get("group", self.group)
-
-        # Add options from RenderGlobals
-        render_globals = instance.data.get("renderGlobals", {})
-        job_info.update(render_globals)
-
-        keys = [
-            "FTRACK_API_KEY",
-            "FTRACK_API_USER",
-            "FTRACK_SERVER",
-            "OPENPYPE_SG_USER",
-            "AYON_BUNDLE_NAME",
-            "AYON_DEFAULT_SETTINGS_VARIANT",
-            "AYON_PROJECT_NAME",
-            "AYON_FOLDER_PATH",
-            "AYON_TASK_NAME",
-            "AYON_WORKDIR",
-            "AYON_APP_NAME",
-            "AYON_IN_TESTS",
-        ]
-
-        environment = {
-            key: os.environ[key]
-            for key in keys
-            if key in os.environ
-        }
-
-        for key in keys:
-            value = environment.get(key)
-            if not value:
-                continue
-            job_info.EnvironmentKeyValue[key] = value
-
-        job_info.add_instance_job_env_vars(self._instance)
-        job_info.add_render_job_env_var()
-
-        # Add list of expected files to job
-        # ---------------------------------
-        if not instance.data.get("multiCamera"):
-            exp = instance.data.get("expectedFiles")
-            for filepath in self._iter_expected_files(exp):
-                job_info.OutputDirectory += os.path.dirname(filepath)
-                job_info.OutputFilename += os.path.basename(filepath)
+        # do not add expected files for multiCamera
+        if instance.data.get("multiCamera"):
+            job_info.OutputDirectory = None
+            job_info.OutputFilename = None
```

Comment:
```suggestion
        # do not add expected files for multiCamera
        # if instance.data.get("multiCamera"):
        #     job_info.OutputDirectory = None
        #     job_info.OutputFilename = None

```
As 3dsMax supports multi camera submission by scene, so maybe we can comment out this one first and we can come back to this again when we refactor the multi-camera submission by cameras in the future.

Reply:
Or maybe resetting it only to ''? It seems to have issue with concatenating None + str

Reply:
> Or maybe resetting it only to ''? It seems to have issue with concatenating None + str
still not working.
It would instead raise `AttributeError: 'str' object has no attribute 'serialize'` in 
```
DEBUG: Using Redshift...published scene wont be used..
INFO: Using D:\sh_boss_proj\interior_lenny\lenny_bedroom\asset\publish\workfile\workfileGeneric\v045\il_asset_workfileGeneric_v045.max for render/export.
DEBUG: Submitting 3dsMax render..
DEBUG: Submitting jobs for multiple cameras..
DEBUG: Executing [GET] settings?project_name=interior_lenny&variant=test_thumbnail-prod-04&bundle_name=test_thumbnail-prod-04&site_id=lumpy-fervent-newt
DEBUG: http://localhost:5000 "GET /api/settings?project_name=interior_lenny&variant=test_thumbnail-prod-04&bundle_name=test_thumbnail-prod-04&site_id=lumpy-fervent-newt HTTP/1.1" 200 113377
DEBUG: Response <RestApiResponse [200]>
Traceback (most recent call last):
  File "C:\Users\Kayla\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "D:\ayon-addon_template\ayon-deadline\client\ayon_deadline\abstract_submit_deadline.py", line 116, in process
    job_id = self.process_submission()
  File "D:\ayon-addon_template\ayon-deadline\client\ayon_deadline\plugins\publish\max\submit_max_deadline.py", line 101, in process_submission
    self.assemble_payload(job_info, plugin_info),
  File "D:\ayon-addon_template\ayon-deadline\client\ayon_deadline\abstract_submit_deadline.py", line 301, in assemble_payload
    "JobInfo": job.serialize(),
  File "D:\ayon-addon_template\ayon-deadline\client\ayon_deadline\lib.py", line 505, in serialize
    serialized.update(attribute.serialize())
AttributeError: 'str' object has no attribute 'serialize'
```

Reply:
Yes - makes sense. You can't assign directly to it like that because you'd be replacing the attribute completely.

You should be doing:
```suggestion
        # do not add expected files for multiCamera
        if instance.data.get("multiCamera"):
            job_info.OutputDirectory.clear()
            job_info.OutputFilename.clear()
```

To just clear the `dict`.

Reply:
> Yes - makes sense. You can't assign directly to it like that because you'd be replacing the attribute completely.
> 
> You should be doing:
> 
> To just clear the `dict`.

Tested with this suggestion, it can successfully render the multi-camera accordingly and also cleans up duplicated the job info in terms of OutputDirectory and OutputFIlename.

Reply:
Ok, I committed that change.